recipes/gridfill/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.3.0" %}
+
+package:
+    name: gridfill
+    version: {{ version }}
+
+source:
+    fn: v{{ version }}.tar.gz
+    url: https://github.com/ajdawson/gridfill/archive/v{{ version }}.tar.gz
+    md5: 4d26437253b77d756e5b37e516c63b51
+
+build:
+    number: 0
+    skip: True  # [win and py3k or win64]
+
+requirements:
+    build:
+        - python
+        - numpy x.x
+        - mingwpy  # [win]
```

Comment:
Ah, then that is probably the issue that @patricksnape was solving. So, I think you could get that to work. Anyways, sorry for the confusion. Thanks for the explanation.